explore/visualization/roc_curves_sample.html

```diff
@@ -0,0 +1,238 @@
+<head>
+  <title>ROC Curves</title>
+  <meta charset="utf-8">
+
+<!-- TODO -->
+<!-- Install with npm install vega -->
+<script src="http://vega.github.io/vega-editor/vendor/d3.min.js" charset="utf-8"></script>

```

Comment:
Let's switch to versioned includes, so we don't have any surprises at deployment time. From https://github.com/vega/vega-lite-demo/issues/1#issuecomment-271972536:

```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js" charset="utf-8"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/vega/2.6.5/vega.min.js" charset="utf-8"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/vega-lite/1.3.1/vega-lite.min.js" charset="utf-8"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/vega-embed/2.2.0/vega-embed.min.js" charset="utf-8"></script>
```